.vib/jaeger/vib-publish.json

```diff
@@ -0,0 +1,81 @@
+{
+  "phases": {
+    "package": {
+      "context": {
+        "resources": {
+          "url": "{SHA_ARCHIVE}",
+          "path": "/bitnami/jaeger"
+        }
+      },
+      "actions": [
+        {
+          "action_id": "helm-package"
+        },
+        {
+          "action_id": "helm-lint"
+        }
+      ]
+    },
+    "verify": {
+      "context": {
+        "resources": {
+          "url": "{SHA_ARCHIVE}",
+          "path": "/bitnami/jaeger"
+        },
+        "runtime_parameters": "cXVlcnk6CiAgc2VydmljZToKICAgIHR5cGU6IExvYWRCYWxhbmNlcgogICAgcG9ydHM6CiAgICAgIGFkbWluOiAzMTAxCiAgICAgIGFwaTogODAKICAgIGNvbnRhaW5lclBvcnRzOgogICAgICBhZG1pbjogMzEwMgogICAgICBhcGk6IDgwOTUKY29sbGVjdG9yOgogIHNlcnZpY2U6CiAgICBwb3J0czoKICAgICAgYWRtaW46IDMxMDEKICAgIGNvbnRhaW5lclBvcnRzOgogICAgICBhZG1pbjogMzEwMgogICAgICBhcGk6IDgwOTUKYWdlbnQ6CiAgc2VydmljZToKICAgIHBvcnRzOgogICAgICBhZG1pbjogMzEwMQogICAgY29udGFpbmVyUG9ydHM6CiAgICAgIGFkbWluOiAzMTAyCiAgICAgIGFwaTogODA5NQogIHBvZFNlY3VyaXR5Q29udGV4dDoKICAgIGZzR3JvdXA6IDEwMDIKICBjb250YWluZXJTZWN1cml0eUNvbnRleHQ6CiAgICBydW5Bc1VzZXI6IDEwMDIKICBzaWRlY2FyczoKICAgIC0gbmFtZTogaG90cm9kLWV4YW1wbGUKICAgICAgaW1hZ2U6IGphZWdlcnRyYWNpbmcvZXhhbXBsZS1ob3Ryb2Q6MS4zOQogICAgICBhcmdzOgogICAgICAgIC0gYWxsCiAgICAtIG5hbWU6IHRyYWNlLWdlbmVyYXRvcgogICAgICBpbWFnZTogYml0bmFtaS9iaXRuYW1pLXNoZWxsOjExLWRlYmlhbi0xMS1yNTcKICAgICAgY29tbWFuZDoKICAgICAgICAtIC9iaW4vYmFzaAogICAgICAgIC0gLWVjCiAgICAgICAgLSB8CiAgICAgICAgICAjIS9iaW4vYmFzaAogICAgICAgICAgd2hpbGUgdHJ1ZTsgZG8KICAgICAgICAgICAgIyBTYW1wbGUgdGFrZW4gZnJvbSBpbnNwZWN0aW5nIGhvdHJvZCBjb2RlCiAgICAgICAgICAgIGN1cmwgJ2h0dHA6Ly8xMjcuMC4wLjE6ODA4MC9kaXNwYXRjaD9jdXN0b21lcj0zOTImbm9uc2U9MC40NTgzODg4NDcxMDUwMTcyNicKICAgICAgICAgICAgc2xlZXAgNQogICAgICAgICAgZG9uZQogIHNlcnZpY2VBY2NvdW50OgogICAgYXV0b21vdW50U2VydmljZUFjY291bnRUb2tlbjogdHJ1ZQpjYXNzYW5kcmE6CiAgZW5hYmxlZDogdHJ1ZQo=",
+        "target_platform": {
+          "target_platform_id": "{VIB_ENV_TARGET_PLATFORM}",
+          "size": {
+            "name": "L4"
```

Comment:
I think that an M4 instance would make it, we could try to reduce it and see how it works